README.rdoc

```diff
@@ -2,6 +2,8 @@
 
 {<img src="https://img.shields.io/gem/v/rubypants.svg?style=plastic">}[https://rubygems.org/gems/rubypants]
 
+{<img src="https://travis-ci.org/jmcnevin/rubypants.svg?branch=master" alt="Build Status" />}[https://travis-ci.org/jmcnevin/rubypants]
```

Comment:
Let's use the shields.io badge with `?style=plastic` so it matches the other one, and get them to render on the same line (however that works in rdoc)